docs/4.2/admin-guide.md

```diff
@@ -2047,6 +2047,10 @@ belong to:
 $ tctl users add joe --k8s-groups="system:masters"
 ```
 
+!!! tip "NOTE"
+
+    The above command is for the Teleport OSS version. Teleport Enterprise customers manage `k8s-groups` through RBAC.
```

Comment:
@aelkugia Could you add a sample role to https://github.com/gravitational/teleport/tree/master/examples/resources

Reply:
Added sample k8s-groups role